### PR TITLE
Hotfix/pd 647 update for prod show view

### PR DESCRIPTION
**Pull request by catherine-jones**
screen shot 2016-10-17 at 10 03 46 pm
screen shot 2016-10-17 at 10 04 34 pm
screen shot 2016-10-17 at 10 04 59 pm
screen shot 2016-10-17 at 10 05 09 pm
screen shot 2016-10-17 at 10 05 17 pm

### CODE REVIEW

**Comment by catherine-jones** [comment]
approved by @phil-banks in previous branch before merge commit hell

File: client/src/dashboard/styles/dashboard/_dashboard.scss [hunk 1]
```diff
@@ -186,6 +186,10 @@
   background-color: $white;
   margin-bottom: 15px;
 
+  @media(max-width: $screen-sm - 1) {
```
Comment (elisechant) [inline client/src/dashboard/styles/dashboard/_dashboard.scss]: i would have flagged this

File: client/src/dashboard/styles/dashboard/_dashboard.scss [hunk 2]
```diff
@@ -321,3 +325,20 @@
     }
   }
 }
+
+.back-link {
```
Comment (elisechant) [inline client/src/dashboard/styles/dashboard/_dashboard.scss]: i would have liked this to be externalised to its own component

File: client/src/dashboard/styles/dashboard/_dashboard.scss [hunk 3]
```diff
@@ -321,3 +325,20 @@
...
+  text-decoration: underline;
+  font-size: 16px;
+  font-size: 1rem;
+  line-height: 1.5;
+  font-weight: 600;
+}
+
+.dashboard-information {
+  .widget.widget-text {
+    height: auto !important; //can be removed when new design implemented
```
Comment (elisechant) [inline client/src/dashboard/styles/dashboard/_dashboard.scss]: this is very stinky. what does this achieve?

**Comment by elisechant** [comment]
@catherine-jones @phil-banks seems to me like we're adding a lot of magic numbers in padding and margins. Can we make a note to externalise a unit of measure for this type of measurements in the near future?

**Pull request merged**